django_project/core/settings/dev_docker.py

```diff
@@ -18,3 +18,4 @@
         'TEST_NAME': 'unittests',
     }
 }
+DOMAIN = 'http://192.168.0.108'
```

Comment:
and here? Lets take these from environment variables or django SITE_URL rather...